test/google_scraping/accounts_test.exs

```diff
@@ -0,0 +1,155 @@
+defmodule GoogleScraping.AccountsTest do
+  use GoogleScraping.DataCase
+
+  import GoogleScraping.AccountsFixtures
+
+  alias GoogleScraping.Accounts
+  alias GoogleScraping.Accounts.User
+  alias GoogleScraping.Accounts.UserToken
+
+  describe "get_user_by_email/1" do
+    test "does not return the user if the email does not exist" do
```

Comment:
How about tp follow our new test description convention? (same as other tests)
```suggestion
    test "When the email does not exist, does not return the user" do
```
![Screen Shot 2022-05-23 at 3 49 59 PM](https://user-images.githubusercontent.com/6076762/169781801-9f6734c1-0fa9-4dcc-9191-fbe315ae3460.png)

Reply:
thank you for pointing on this option!